shelley/design-spec/delegation_design_spec.tex

```diff
@@ -248,9 +240,15 @@ \subsubsection{Handle Inactive Stake Pools}
 \label{handle-inactive-stake-pools}
 
 We anticipate that a stake pool operator can cease to operate -- whether
-they lost their keys, lost interest, died, etc. We want to minimise the
```

Comment:
As of this PR, stake pools operators are not allowed to die anymore :P I understand the change though ...

Reply:
I was told it was a bit morbid :)